editor/edit-post/header/plugins/index.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import { map, isEmpty } from 'lodash';
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { NavigableMenu, withInstanceId, IconButton } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { getEllipsisMenuItems } from '../../../api/ellipsis-menu';
+
+/**
+ * Renders a list of plugins that will activate different UI elements.
+ *
+ * @param   {Object} props The component props.
+ * @returns {Object} The rendered list of menu items.
+ */
+function Plugins( props ) {
+	const ellipsisMenuItems = getEllipsisMenuItems();
```

Comment:
Worth noting that, by sourcing available items this way, `Plugins` won't re-render if a menu item is registered dynamically after init.

Reply:
The question is whether you want the ellipsis menu to be updated during runtime? This could lead to some bad user experience.